File: Modul-6/Soal 5/PRAK605-2410817110004-MuhammadNaufalKhalish.c
#include <stdio.h>
#include <string.h>

int main(){
    int ordo;
    scanf("%d",&ordo);

    int matrikspertama[ordo][ordo];
    int matrikskedua[ordo][ordo];
    int hasil[ordo][ordo];
    char awalnama1[100];
    char namamatriks1[100];
    char awalnama2[100];
    char namamatriks2[100];

    getchar();
    scanf("%s %s",&awalnama1,&namamatriks1);
    namamatriks1[strcspn(namamatriks1, "\n")]= '\0';

    for(int baris=0; baris<ordo; baris++){
        for(int kolom=0; kolom<ordo; kolom++){
            scanf("%d",&matrikspertama[baris][kolom]);
            }
        }
     getchar();

    scanf("%s %s",&awalnama2,&namamatriks2);
    namamatriks2[strcspn(namamatriks2, "\n")]= '\0';

    for(int baris=0; baris<ordo; baris++){
        for(int kolom=0; kolom<ordo; kolom++){
            scanf("%d",&matrikskedua[baris][kolom]);
        }}

    for(int i=0; i<ordo; i++){
    for(int j=0; j<ordo; j++){
        hasil[i][j]=0;
    for(int k=0; k<ordo; k++){
        hasil[i][j]+=matrikspertama[i][k]*matrikskedua[k][j];
        } } }

    printf("\n");
    printf("Matriks %sX%s \n",namamatriks1,namamatriks2);
    for(int i=0;i<ordo; i++){
        for(int j=0; j<ordo; j++){
            printf("%d ",hasil[i][j]);
        } printf("\n");
    }
    return 0; }

File: Modul-6/Soal 5/PRAK605-2410817110004-MuhammadNaufalKhalish.py
ordo=int(input())

awalnama1,namamatriks1=input().split()
matrikspertama=[]
for i in range(ordo):
    baris=list(map(int,input().split()))
    matrikspertama.append(baris)

awalnama2,namamatriks2=input().split()
matrikskedua=[]
for i in range(ordo):
    baris2=list(map(int,input().split()))
    matrikskedua.append(baris2)

hasil= [[0] * ordo for _ in range(ordo)]

for i in range(ordo): 
    for j in range(ordo):
        for k in range(ordo):
            hasil[i][j]+=matrikspertama[i][k]*matrikskedua[k][j]

print(f"Matriks {namamatriks1}X{namamatriks2}")
for baris in hasil:
        print(" ".join(map(str, baris)))
